homeassistant/components/fritz/common.py

```diff
@@ -184,6 +184,14 @@ def _fetch_device_info(self):
         return dev_info
 
 
+class FritzData:
```

Comment:
Any reason to make this a class and not just store a dict in `hass.data[DATA_FRITZ]` ?

Reply:
I thought it might be a convenient class for future extensions of the component. Should I change it @balloob?
Thanks for your review! I will address the changes soon!

Reply:
Ok, works for me.

Reply:
This could be a dataclasses.dataclass.

https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass